### PR TITLE
fix parseChar see #16240

### PR DESCRIPTION
**Pull request by pietroppeter**
fix minor issue in recent PR by @narimiran (#16240), see:
https://github.com/nim-lang/Nim/pull/16240/files#r535452099

### CODE REVIEW

File: lib/pure/parseutils.nim [hunk 1]
```diff
@@ -211,6 +211,7 @@ proc parseHex*[T: SomeInteger](s: string, number: var T, start = 0,
 proc parseIdent*(s: string, ident: var string, start = 0): int =
   ## Parses an identifier and stores it in ``ident``. Returns
   ## the number of the parsed characters or 0 in case of an error.
+  ## If error, the value of ``ident`` is not changed.
```
Comment (ringabout) [inline lib/pure/parseutils.nim]: Maybe use single quote which is more simple.
```nim
If error, the value of `ident` is not changed.
```

File: lib/pure/parseutils.nim [hunk 2]
```diff
@@ -241,9 +242,19 @@ proc parseIdent*(s: string, start = 0): string =
     while i < s.len and s[i] in IdentChars: inc(i)
     result = substr(s, start, i-1)
 
-proc parseChar*(s: string, ident: var char, start = 0): int =
-  ident = s[start]
-  result = 1
+proc parseChar*(s: string, c: var char, start = 0): int =
+  ## Parses a single character, stores it in ``c`` and returns 1.
+  ## In case of error (if start >= s.len) it returns 0
+  ## and the value of ``c`` is unchanged.
```
Comment (ringabout) [inline lib/pure/parseutils.nim]: same as above: use single quote?

File: lib/pure/parseutils.nim (hunk 1 above)
Comment (pietroppeter) [reply lib/pure/parseutils.nim]: I can definitely do that, but then I would change it throughout the document (lots of other places where double backticks appear) to keep it consistent. should I do that?

Comment (ringabout) [reply lib/pure/parseutils.nim]: Yeah, two way exists. You don't have to clean them in this PR, just not introducing them is enough.
See https://github.com/timotheecour/Nim/issues/415

Comment (pietroppeter) [reply lib/pure/parseutils.nim]: done

**Pull request merged**